html-block-and-inline/html-block-and-inline-quiz-notes.md

```diff
@@ -0,0 +1,42 @@
+# html-block-and-inline-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- How do block-level elements affect the document flow?
+  /it typically starts on a new line and stretches out to the full width available
+- How do inline elements affect the document flow?
+  /it does not start on a new line and only occupy the space bounded by the tags defining the element
+- What are the default width and height of a block-level element?
+  /the default height is the height of the content and the width is the length of the page
```

Comment:
note that width is the full width available to it based on its parent, not necessarily the whole page...